modules/executors/main.tf

```diff
@@ -123,6 +123,7 @@ resource "google_compute_firewall" "executor-access" {
     protocol = "tcp"
     ports = [
       "6060" # Debug server
+      "22"   # ssh
```

Comment:
Should we maybe for feature parity catch up with what we do in AWS and make both things configurable and allow separate CIDR ranges? https://github.com/sourcegraph/terraform-aws-executors/blob/master/modules/executors/main.tf#L48-L64